medcat/utils/meta_cat/data_utils.py

```diff
@@ -61,21 +65,39 @@ def prepare_from_json(data: Dict,
                                 break
 
                     if not skip and (cui_filter is None or not cui_filter or cui in cui_filter):
-                        if ann.get('validated', True) and (not ann.get('deleted', False) and not ann.get('killed', False)
-                                                           and not ann.get('irrelevant', False)):
+                        if ann.get('validated', True) and (
+                                not ann.get('deleted', False) and not ann.get('killed', False)
+                                and not ann.get('irrelevant', False)):
                             start = ann['start']
                             end = ann['end']
 
                             # Get the index of the center token
+                            flag = 0
```

Comment:
Again, looks like some kind of new behaviour. Why is this added? Will this retain old behaviour?

Reply:
Its the same change: it changes the way we extract the tokens for the medical entity in question.